docs/tutorials

```diff
@@ -0,0 +1 @@
+C:/Users/ChristiaanMeijer/Documents/dianna/dianna/tutorials
```

Comment:
I'm pretty sure this won't work apart from on Christiaan's machine ;) 
We could use nblink instead?

Reply:
haha, I can see why you'd be suspicious about that line, but have you tried running it? ;-P

Anyway, we hoped to fix this with a symlink (like we tried here) but my experience with symlinks on Linux is already limited but on Windows even less. I guess this was my first attempt to make one on Windows.
We choose not to use nbsphinx_link because that would mean we'd need to do bookkeeping of our notebooks twice, which is errorprone. Also, because the package was not maintained the last 2 years, we preferred to not rely on it.

I think I'll try to fix the symlink first. If that doesn't work, we can switch to nbsphinx_link anyway.

Reply:
Apparently making a relative link to a directory is some cutting edge feature that my current Powershell cannot handle yet. I'll check if I can get my hands on some newer version later (tomorrow probably) to test that out. I have PS5 and apparently I'd need PS7.1 (https://stackoverflow.com/questions/42513791/how-to-create-relative-symlink-powershell-way)

Reply:
Meine Güte. I'm reading up on teh Googlez a bit and it seems symlinks in git on Windows in general is all still iffy and annoying (needs special rights, maybe git settings, git hooks...). I'm starting to wonder whether it's worth the effort. Maybe revert to using `nbsphinx_link` after all?

Reply:
Unfortunately the windows-type relative symlink doesn't work on Mac/Linux. We only needs this to build the documentation anyway, right? We could limit the doc build action to only one OS and use that to deploy it. But probably going to nbsphinx_link is easiest...

Reply:
That is a good idea as well! But I think I agree that using nbsphinx_link is slightly better, since that way Windows devs can also still build docs locally.

Reply:
I managed to create a symlink in PS7.1. It clearly shows it's there. But even then, I haven't even figured out how to use it, it keeps showing up in listings as a link, without showing what's behind it. It's a pain really. Shame I only read your comments just now, otherwise I probably would have stopped trying an hour ago.

Anyway, I'm now going for the nbsphinx_link approach again. Thanks for your thoughts and suggestions.